relay-server/src/services/processor.rs

```diff
@@ -2041,13 +2049,9 @@ impl EnvelopeProcessorService {
             let limits = project_state.get_cardinality_limits();
 
             let buckets = self.cardinality_limit_buckets(scoping, limits, buckets, mode);
+            let quotas = DynamicQuotas::new(&global_config, &project_state.config.quotas);
```

Comment:
```suggestion
            let quotas = DynamicQuotas::new(&global_config, project_state.get_quotas());
```

You use `get_quotas` elsewhere, might as well do it here.